keyboards/1upkeyboards/1up60hte/1up60hte.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2019 Bubnick
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
```suggestion
#define LAYOUT( \
```
Layout macros should have names starting with `LAYOUT`. You will need to update usages in `keymap.c` as well. If you want this board to be compatible with shared community layouts, you'll need to define `LAYOUT_all` and other layouts that this PCB supports.
For more details, see how it's done in other 1upkeyboards boards, for example 1up60rgb.